packages/frontend/src/components/Trade/Long/BreakEven.tsx

```diff
@@ -0,0 +1,37 @@
+import useAppMemo from '@hooks/useAppMemo'
+import { getBreakEvenForLongSqueeth, toTokenAmount } from '@utils/calculations'
+import { useAtomValue } from 'jotai'
+import React from 'react'
+import { currentImpliedFundingAtom, indexAtom, markAtom, normFactorAtom } from 'src/state/controller/atoms'
+import TradeInfoItem from '../TradeInfoItem'
+
+const BreakEven: React.FC = () => {
+  const mark = useAtomValue(markAtom)
+  const index = useAtomValue(indexAtom)
+  const days = 1
+  const normFactor = useAtomValue(normFactorAtom)
+  const currentFunding = useAtomValue(currentImpliedFundingAtom)
+
+  const breakEven = useAppMemo(() => {
+    const breakEvenValue = getBreakEvenForLongSqueeth(mark, index, normFactor, days)
+    const ethPrice = toTokenAmount(index, 18).sqrt()
+    return (breakEvenValue / ethPrice.toNumber() - 1) * 100
+  }, [index, mark, normFactor])
+
+  return (
+    <div>
+      <TradeInfoItem
+        label="ETH move to breakeven"
+        value={breakEven.toFixed(3)}
+        tooltip={`To breakeven ETH to needs to move to ${breakEven.toFixed(
+          3,
+        )}% in one day, since current funding is{' '}
+        ${(currentFunding * 100).toFixed(2)}%`}
```

Comment:
Looks like smth slightly off with the formatting here
<img width="354" alt="Screen Shot 2022-05-10 at 9 50 01 PM" src="https://user-images.githubusercontent.com/13340486/167771674-c4e1313f-49d6-4bae-bee9-03f49322eb69.png">